backend/db/migrations/j12bouy3.age-range.sql

```diff
@@ -0,0 +1,50 @@
+ALTER TABLE annotation_attribute
+  ADD type TEXT;
+
+UPDATE annotation_attribute
+  SET type = 'select-one'
+  WHERE NOT(name = 'Perceived Age');
```

Comment:
Just calling out that you can do `WHERE name != 'Perceived Age'` No need to change it, I just find that syntax a little more readable.